packages/sanity/src/desk/panes/document/timeline/timelineItem.tsx

```diff
@@ -43,9 +41,7 @@ export function TimelineItem({
 }: TimelineItemProps) {
   const iconComponent = getTimelineEventIconComponent(type)
   const authorUserIds = Array.from(chunk.authors)
-  const isSelectable = useMemo(() => {
-    return !NOT_SELECTABLE_TYPES.includes(type)
-  }, [type])
```

Comment:
**question:** what's the rationale behind making 'discarded draft' revs non selectable?

**suggestion:** we allow discarded drafts to be selectable as it was prior – there are cases where users may have a discarded draft as their most recent version, and will need to select it to return to the most up to date / recent version

Making deleted revisions non-selectable is a great shout though!

Reply:
Great point, I believed it was similar to the deleted event, and thought it didn't make sense to select it.